### PR TITLE
HCAP-1271: 'Move to archive' bug fix

### PR DESCRIPTION
**Pull request by alva-fresh**
https://freshworks.atlassian.net/browse/HCAP-1271

HA are now able to move withdrawn participants in `hired` state to archive

### CODE REVIEW

File: client/src/pages/private/ParticipantTable.js [hunk 1]
```diff
@@ -233,7 +233,7 @@ const ParticipantTable = () => {
     setActiveModalForm(participantEngageStatus.MULTI_SELECT_SITE);
   };
 
-  const handleEngage = async (participantId, status, additional = {}, participantInfo = null) => {
+  const handleEngage = async (participantId, status, additional = {}, _participantInfo = null) => {
```
Comment (arranfw) [inline client/src/pages/private/ParticipantTable.js:236]: Can we just remove this parameter?

**Review by arranfw — COMMENTED** [review]
I think this makes sense, I'm mostly just not sure about which status' are set on which actions

What status does MoH archival put the user into? Based on this change I would think `hired` but that doesn't make sense

What are the statuses when the HA is archiving the user? If they're archived by MoH why are they being archived again?

File: client/src/pages/private/ParticipantTable.js (hunk 1 above)
Comment (alva-fresh) [reply client/src/pages/private/ParticipantTable.js:236]: I wonder the same, not sure at what stage it became optional. But it's being passed on for every `handleEngage`

**Comment by fw-noel** [comment]
We need Tavy to confirm what behavior should look like here

**Pull request closed without merging**